Calmspace/DetailsTableViewController.swift

```diff
@@ -0,0 +1,152 @@
+//
+//  DetailsTableViewController.swift
+//  Calmspace
+//
+//  Created by Admin on 12/10/2023.
+//
+
+import UIKit
+import Foundation
+
+struct PracticeDetail {
+    let title: String
+    let url: URL
+}
+
+
+
+class DetailsTableViewController: UITableViewController {
+    
+    
+    var selectedArray: [PracticeDetail] = []
+    
+    
+    
+    // Assuming you have the practiceData instance available
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        
+        
+    }
+    
+    
+    // Uncomment the following line to preserve selection between presentations
+    // self.clearsSelectionOnViewWillAppear = false
+    
+    // Uncomment the following line to display an Edit button in the navigation bar for this view controller.
+    // self.navigationItem.rightBarButtonItem = self.editButtonItem
+    
+    
+    // MARK: - Table view data source
+    
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        // #warning Incomplete implementation, return the number of sections
```

Comment:
do we need this warning? Is it incomplete? If not, the warning should be removed here and in other places